File: backend/models/journal.js
import mongoose from 'mongoose';

const journalSchema = new mongoose.Schema({
    title:{
        type: String,
        required: true
    },
    mood:{
        type: String,
        required: true
    },
    input:{
        type: String,
        required: true
    },
}, {
    timestamps: true
});

const journal = mongoose.model('journal', journalSchema);

export default journal;

File: frontend/lib/models/journal.dart
import 'package:objectbox/objectbox.dart';

@Entity()
class Journal {
  int id;
  int loginId;
  String title;
  String mood;
  String intensity;
  String time;
  String journal;

  Journal({
    this.id = 0,
    required this.loginId,
    required this.title,
    required this.mood,
    required this.intensity,
    required this.time,
    required this.journal
  });
}